.asyncapi-tool

```diff
@@ -0,0 +1,12 @@
+title: Java Spring Template
+filters:
+  language: 
+  - Java
```

Comment:
@TheEques `language` is an info in what language the project is written. In case of generator templates it is `javascript`, even thought Java is generated, the template sources are in JS

Reply:
In the code section of this repo it showed Java~68% and javascript~31% hence I thought I should add both
Should I change it? @derberg

Reply:
maybe it analyses snapshot tests where java code is. If you can open followup that would be great

Reply:
I think GitHub just takes file extensions in repo and most of them are `.java` :)